### PR TITLE
invert bit in javascript

### PR DESCRIPTION
**Pull request by tbhaxor**
**Fixes issue:** NONE

**Changes:** invert bit in javascript added

### CODE REVIEW

File: code/bit_manipulation/src/invert_bit/invert_bit.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const invertBit = (num, bit) => num ^= 1 << bit;
```
Comment (arnavb) [inline code/bit_manipulation/src/invert_bit/invert_bit.js]: Why the assignment to `num`?

Comment (tbhaxor) [reply code/bit_manipulation/src/invert_bit/invert_bit.js]: idk but code is working :laughing:

Comment (arnavb) [reply code/bit_manipulation/src/invert_bit/invert_bit.js]: Does it work if you replace `^=` with `^`?

Comment (tbhaxor) [reply code/bit_manipulation/src/invert_bit/invert_bit.js]: no, wrong answer is coming ..

lets see what @AdiChat says

Comment (AdiChat) [reply code/bit_manipulation/src/invert_bit/invert_bit.js]: @tbhaxor This is related to JavaScript syntax. :+1: 

If you do not use the `return` keyword, then you need to assign the return value to some variable to achieve the same functionality. Therefore, if you want to remove `=` then, you must use `return` keyword.

Comment (arnavb) [reply code/bit_manipulation/src/invert_bit/invert_bit.js]: @AdiChat are you sure about that? As far as I know, [ES6 arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) without braces have an implicit return, so the explicit keyword isn't necessary. @tbhaxor I don't see an incorrect answer; When I remove the `=`, then I still get the same result. This is the code I am using (It's a diff):

```diff
-const invertBit = (num, bit) => num ^= 1 << bit;
+const invertBit = (num, bit) => num ^ 1 << bit;

console.log(invertBit(10, 5));  // 42
```

And here is a [JS Bin](http://jsbin.com/nofanizawo/edit?js,console) that illustrates a working example.

Comment (AdiChat) [reply code/bit_manipulation/src/invert_bit/invert_bit.js]: Yes, you are right. It looks like I was thinking of functions. :sweat_smile: 

I will reflect the change and merge this. :+1:

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

positive_opengenus_cosmos_1

**Comment by arnavb** [comment]
Nice image choice! (Do you make an image every time, for the username?)

**Comment by AdiChat** [comment]
Yes, I make a custom image for every user :smile:

**Comment by arnavb** [comment]
😮

**Pull request merged**